accounting/bank/feeds/ponto.rst

```diff
@@ -0,0 +1,115 @@
+   moving on to the next steps.
+
+.. _ponto-link-odoo:
+
+Link your Ponto account with your Odoo database
+-----------------------------------------------
+
+Odoo requires you to insert your Ponto's **Client ID** and **Secret ID** to synchronize both
+platforms.
+
+.. image:: media/ponto_link_odoo.png
+   :align: center
+   :alt: Logo of the Ponto brand
+
+#. To generate these IDs, go to :menuselection:`Ponto Dashboard --> Integration --> Live`, click on
+   **Add Integration**, fill out the form, and select which accounts you want to synchronize.
+#. Copy the *Client ID* and *Secret ID* generated and paste them in Odoo.
+#. | Configure the synchronization options:
+   | - *Action*: define if Odoo must create a new *Journal* for this bank account, link to an
+     existing Journal, or create no link.
+   | - *Journal*: select the appropriate *Journal*.
+   | - *Synchronization Frequency*: define how often Odoo should fetch the bank statements.
+
+.. image:: media/ponto_synchronization.png
+   :align: center
+   :alt: Configure the bank synchronization through Ponto on your Odoo database
+
+.. tip::
+   It is good practice to have one Journal per bank account. If you synchronize a single bank
+   account, link it to the existing *Bank* journal. If you have multiple accounts, it is recommended
+   to pick the *Create new journal* option for all additional accounts.
+
+.. _ponto-update-credentials:
+
+Update your synchronization credentials
+---------------------------------------
+
+You might have to update your Ponto credentials or modify the synchronization settings.
+
+To do so, activate the :doc:`Developer mode <../../../general/developer_mode/activate>`, and go to
+:menuselection:`Accounting --> Configuration --> Online Synchronization`.
+
+Click on *Update Accounts* to enter your new *Client ID* and *Secret ID*, or click on *Edit* to
+modify the synchronization settings.
+
+Deprecated API tokens
+=====================
+
+Ponto's previous synchronization system using API tokens is now deprecated.
+
+This section is only relevant for users who had previously linked Ponto with Odoo using a single API
+token instead of the current synchronization system with a *Client ID* and a *Secret ID*.
+
+#. | **Update** your database.
+   | - *SaaS* and *Odoo.sh* users: you can skip this step, as your database is automatically updated.
+   | - *Community* and *Enterprise* users: download the latest Odoo source code for your version
+     (you can download it from `this link <https://odoo.com/download>`_ or from GitHub), install it,
+     and restart your server.
+#. | Do a **hard refresh** of your Odoo page by pressing *CTRL + F5*.
+   | A hard refresh clears the cache and the javascript code for the current page.
+#. To **generate your access key**, follow :ref:`these steps above <ponto-link-odoo>`.
+#. To **update your credentials**, follow :ref:`these steps above <ponto-update-credentials>`.
```

Comment:
```suggestion
#. To **update your credentials**, follow :ref:`the steps above <ponto-update-credentials>`.
```

Reply:
Same as above. I used the line break to keep the main info apart from the explanation because we are in a list.